### PR TITLE
Jmprieur/webappwebapib2c

### PR DESCRIPTION
**Pull request by jmprieur**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by jennyf19** [comment]
> using Microsoft.AspNetCore.Authorization;

header info

---
Refers to: 4-WebApp-your-API/4-2-B2C/Client/Controllers/HomeController.cs:1 in f07deb6. [](commit_id = f07deb6644a9caa56bda0fc93fb8a400c5268a63, deletion_comment = False)

**Comment by jennyf19** [comment]
> using Microsoft.AspNetCore.Http;

header info

---
Refers to: 4-WebApp-your-API/4-2-B2C/Client/Controllers/TodoListController.cs:1 in f07deb6. [](commit_id = f07deb6644a9caa56bda0fc93fb8a400c5268a63, deletion_comment = False)

File: 4-WebApp-your-API/4-2-B2C/Client/Infrastructure/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+namespace WebApp_OpenIDConnect_DotNet.Infrastructure
```
Comment (jennyf19) [inline 4-WebApp-your-API/4-2-B2C/Client/Infrastructure/Constants.cs]: header info....do we even use these?

File: 4-WebApp-your-API/4-2-B2C/Client/Models/ErrorViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+namespace WebApp_OpenIDConnect_DotNet.Models
```
Comment (jennyf19) [inline 4-WebApp-your-API/4-2-B2C/Client/Models/ErrorViewModel.cs]: header info

**Comment by jennyf19** [comment]
> using System;

header info

---
Refers to: 4-WebApp-your-API/4-2-B2C/TodoListService/Models/TodoItem.cs:1 in f07deb6. [](commit_id = f07deb6644a9caa56bda0fc93fb8a400c5268a63, deletion_comment = False)

File: 4-WebApp-your-API/4-2-B2C/Client/appsettings.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    "ClientId": "fdb91ff5-5ce6-41f3-bdbd-8267c817015d",
+    "Domain": "fabrikamb2c.onmicrosoft.com",
+    "SignedOutCallbackPath": "/signout/B2C_1_susi",
+    "SignUpSignInPolicyId": "b2c_1_susi",
+    "ResetPasswordPolicyId": "b2c_1_reset",
+    "EditProfilePolicyId": "b2c_1_edit_profile", // Optional profile editing policy
+    "ClientSecret": "X330F3#92!z614M4"
+    //"CallbackPath": "/signin/B2C_1_sign_up_in"  // defaults to /signin-oidc
+  },
+  "AzureAd": {
```
Comment (TiagoBrenck) [inline 4-WebApp-your-API/4-2-B2C/Client/appsettings.json]: Why do we need AzureAD section in this B2C appsettings?

Comment (jmprieur) [reply 4-WebApp-your-API/4-2-B2C/Client/appsettings.json]: We don't. This is merged in RemoveUIs, we should remove it from there.

**Pull request merged**